src/apps/js_generic/js_generic.cpp

```diff
@@ -14,7 +14,7 @@ namespace ccfapp
 
   std::vector<ccf::js::FFIPlugin> get_js_plugins()
   {
-#if defined(INSIDE_ENCLAVE) || defined(SGX_ATTESTATION_VERIFICATION)
+#if defined(INSIDE_ENCLAVE) && defined(SGX_ATTESTATION_VERIFICATION)
     return {ccf::js::openenclave_plugin};
```

Comment:
The INSIDE_ENCLAVE check here seems slightly misleading, because:

1. We don't create interpreters outside an INSIDE_ENCLAVE context
2. There is no reason this shouldn't work from host code if oehost_verify is linked

so I think we should just check SGX_ATTESTATION_VERIFICATION